roles/mesos/tasks/follower.yml

```diff
@@ -78,6 +78,15 @@
   tags:
     - mesos
 
+- name: set follower resources
+  sudo: yes
+  copy:
+    dest: /etc/mesos-slave/resources
+    content: "{{ mesos_resources }}"
+  notify: restart mesos follower
```

Comment:
The only other task that uses that handler is the follower node ID below. I'm pretty sure that's stored in `/tmp/mesos/meta/slaves/latest`, but it'd be worth grepping around to see if you can find where it's stored. If it's in the longer path, changing the handler would probably be the best course of action.